text/0000-dot-node-version-file-support.md

```diff
@@ -0,0 +1,63 @@
+- Feature Name: dot_node_version_file_support
+- Start Date: 2025-03-04
+- RFC PR: (leave this empty)
+- Volta Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Introduce support for a `.node-version` file, enabling it as a fallback source for determining project-specific Node versions. If node version is not specified in `package.json`, fallback to `.node-version` before using global defaults.
+
+# Motivation
+[motivation]: #motivation
+
+The aim is to align Volta with other popular Node version managers like fnm, which support `.node-version`. This enhances flexibility and user experience by allowing developers to specify their preferred Node versions in a widely recognized format.
+
+# Pedagogy
+[pedagogy]: #pedagogy
+
+Existing node developers are likely already familiar with other versioning methods like `.nvmrc`. Introducing support for `.node-version` aligns Volta with other node version managers.
+
+# Details
+[details]: #details
+
+## Inheritance from current solution
+
+The new feature will follow the same logic currently applied when reading a node version from `package.json`. This ensures consistency across different configuration files.
+
+## Standardized Approach
+
+This implementation should mirror the behavior of other Node version managers, ensuring Volta remains competitive and user-friendly.
+
+## Fallback Hierarchy
+
+Prioritize `.node-version` as fallback for determining Node versions.
+If volta is not defined in `package.json`, check `.node-version` before proceeding with global defaults.
+Continue with existing behavior if neither file specifies a version.
```

Comment:
4/5 (serious issue: blocking)

As written, this _feels_ backwards. It would be better to actually define the lookup precedence directly (instead of talking in terms of fallbacks).

Something like (but with better/more words):

```
The Node version lookup process will be:

1. Resolve the `package.json` and check it's `volta.node` field
  - if present, use that explicit node version
2. Check for the presence of `.node-version`
  - if present, use the value specified
3. Fallback to the default active `node` toolchain
```